src/AnalyzerPowerPack/Core/Usage/SerializationRulesDiagnosticAnalyzer.cs

```diff
@@ -126,17 +132,15 @@ private sealed class Analyzer
             private readonly INamedTypeSymbol _serializationInfoTypeSymbol;
             private readonly INamedTypeSymbol _streamingContextTypeSymbol;
             private readonly INamedTypeSymbol _serializableAttributeTypeSymbol;
+            private readonly INamedTypeSymbol _nonSerializedAttributeTypeSymbol;
 
-            public Analyzer(
-                INamedTypeSymbol iserializableTypeSymbol,
-                INamedTypeSymbol serializationInfoTypeSymbol,
-                INamedTypeSymbol streamingContextTypeSymbol,
-                INamedTypeSymbol serializableAttributeTypeSymbol)
+            public Analyzer(INamedTypeSymbol iserializableTypeSymbol, INamedTypeSymbol serializationInfoTypeSymbol, INamedTypeSymbol streamingContextTypeSymbol, INamedTypeSymbol serializableAttributeTypeSymbol, INamedTypeSymbol nonSerializedAttributeTypeSymbol)
```

Comment:
@hvanbakel Given the length of this line I would prefer it be split across multiple lines, as it was before.